site/components/src/main/java/uk/nhs/digital/website/beans/Video.java

```diff
@@ -0,0 +1,26 @@
+package uk.nhs.digital.website.beans;
+
+import org.hippoecm.hst.content.beans.Node;
+import org.hippoecm.hst.content.beans.standard.HippoBean;
+import org.hippoecm.hst.content.beans.standard.HippoHtml;
+import org.onehippo.cms7.essentials.dashboard.annotations.HippoEssentialsGenerated;
+
+import java.util.List;
+
+@Node(jcrType = "website:video")
+public class Video extends CommonFieldsBean {
+    @HippoEssentialsGenerated(internalName = "website:introduction", allowModifications = false)
+    public HippoHtml getIntroduction() {
+        return getHippoHtml("website:introduction");
+    }
+
+    @HippoEssentialsGenerated(internalName = "website:externallinkbase")
+    public String getVideoUri() {
+        return getSingleProperty("website:externallinkbase");
```

Comment:
The property 'website:externallinkbase' should also be in camel case ('website:externalLinkBase'). The other files will also need updating to change of case.